### ISSUE TITLE
Accuracy of VN_DGCNN on the ModelNet40

### ISSUE THREAD

**Issue by AkonLau** [FlyingGiraffe/vnn#6]
Hi, 
Thanks for your work and codes. However, when I reproduce your code with the VN_DGCNN model and default parameters, I just got 80.2% in the I/I case and 78.8% in the z/z case. The results are much less than the accuracy reported in the paper. 
Are there any details I missed? Could you help us analyze the problem or give more details for discussion? 
Thanks a lot!

**Comment by FlyingGiraffe** [comment]
Hi, thanks for the comment!

I was using the [DGCNN](https://github.com/WangYueFt/dgcnn) code framework when generating the results in the paper, but later when publishing the code I merged the models into the [PointNet](https://github.com/yanx27/Pointnet_Pointnet2_pytorch) framework for convenience. I quickly skimmed through the code and one reason might be: in DGCNN (and thus same for VN_DGCNN in the paper), the optimizer is SGD with`lr=0.1` (see [here](https://github.com/WangYueFt/dgcnn/blob/e96a7e26555c3212dbc5df8d8875f07228e1ccc2/pytorch/main.py#L61)), but the default optimizer here is Adam with `lr=0.001`.

I'll double-check with other settings. Thanks again for noticing this!

Best,
Congyue

**Comment by GostInShell** [comment]
Thanks for the wonderful code! With the above modification, I achieve the results 73.4% on SO3/SO3. Moreover, I find the training process rather unstable with large fluctuation, comparing to the normal DGCNN and PointNet. Is this usual? If possible, could you please share the training log?
